src/main/java/org/xstefank/model/Utils.java

```diff
@@ -59,4 +79,8 @@ public static URL getFormatUrl() throws MalformedURLException {
             target = getTyrProperty(TEMPLATE_FORMAT_URL);
         return target != null ? new URL(target) : null;
     }
+
+    public static boolean getBooleanProperty(String key) {
```

Comment:
maybe add also getIntProperty? you were parsing integer from getTyrProperty as port number

Reply:
Ok I see what you mean. But it makes no sense to me write there that method because I passed all the properties to TeamCityCI class constructor. So the method would not be used anywhere for this case. Maybe for next force push it would...